Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/BypassDampers.mo

```diff
@@ -0,0 +1,208 @@
+within Buildings.Fluid.HeatExchangers.ThermalWheels.Latent;
+model BypassDampers
+  "Enthalpy recovery wheel with bypass dampers"
+  extends
+    Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.PartialWheel;
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput uBypDamPos(
+    final unit="1",
+    final min=0,
+    final max=1)
+    "Bypass damper position"
+    annotation (Placement(transformation(extent={{-222,120},{-182,160}}),
+      iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uRot
+    "True when the wheel is operating"
+    annotation (Placement(transformation(extent={{-220,-20},{-180,20}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Supply air bypass damper"
+    annotation (Placement(transformation(extent={{-60,70},{-40,90}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Supply air damper"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},rotation=0,origin={-50,40})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
```

Comment:
It is unreasonable -- and wrong -- to set the damper pressure drops to the same value as the heat exchanger pressure drop, which is by default 500 Pa.

Reply:
@SenHuang19 Has it been addressed?

Reply:
I believe so. The damper pressure is now a separate parameter, set to 20 Pa by default.